src/main/java/lotto/context/Rank.java

```diff
@@ -0,0 +1,33 @@
+package lotto.context;
+
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(6, 2000000000),
```

Comment:
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요!
```suggestion
    FIRST(6, 2_000_000_000),
```